packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -113,6 +113,24 @@ export const deleteDraftDescriptorErrorMapper = (
     .with("operationForbidden", () => HTTP_STATUS_FORBIDDEN)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
+export const updateDraftDescriptorErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with(
+      "eServiceNotFound",
+      "eServiceDescriptorNotFound",
+      () => HTTP_STATUS_NOT_FOUND
+    )
+    .with("operationForbidden", () => HTTP_STATUS_FORBIDDEN)
+    .with(
+      "notValidDescriptor",
+      "inconsistentDailyCalls",
+      "attributeNotFound",
+      () => HTTP_STATUS_BAD_REQUEST
+    )
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
```

Comment:
Currently `updateDraftDescriptor` and `updateDescriptor` were using the same error mapper because they were throwing the same errors. With the change in this PR, now `updateDraftDescriptor` also throws `attributeNotFound`, so we could distinguish the two error mappers.